### ISSUE TITLE
Untagged articles do not get included

### ISSUE THREAD

**Issue by anjuthanmj** [mmagnus/Pocket-Plus-Calibre-Plugin#60]
I'm trying to use the plugin for first time and not able to make it work. I the the error No articles in the pocket account <my-email> to download. How can I rectify this?

![image](https://github.com/user-attachments/assets/2b2e15bd-15c9-4205-b156-48ea549c6966)

**Comment by alvaroreig** [comment]
The problem might be in several params:

OLDEST_ARTICLE
TO_PULL
TAGS_EXCEPTIONS
URL_KEYWORD_EXCEPTIONS

I would verify them. If the problems persist, paste the values of all params and some examples of the articles you are trying to download.

**Comment by JosemyDuarte** [comment]
I'm having the same problem. Noticed the following couple of behaviors:

With a clean account (everything else archived), I add a URL to Pocket and don't set any tag, it shows the problem described (no article found).
If I set any tag to the article, it works as expected.
If I choose to sort the articles by TLD then it works as expected.

**Comment by anjuthanmj** [comment]
@alvaroreig These are my configuration values. 

# CONFIGURATION ###########################################################
TAGS = [] # [] or ['tag1', 'tag2']
TAGS_EXCEPTIONS = [] # [] or ['tag3', 'tag4']
URL_KEYWORD_EXCEPTIONS = [] # [] or ['keyword1', 'keyword2']
SECTIONS_BY_DOMAIN = False
INCLUDE_UNTAGGED = True
ARCHIVE_DOWNLOADED = True
MAX_ARTICLES = 1000
OLDEST_ARTICLE = 365
SORT_METHOD = 'newest'
SORT_WITHIN_TAG_BY_TITLE = False
TO_PULL = 'unread'
TITLE_WITH_TAGS = False
ALLOW_DUPLICATES = True
USE_GLOBAL_CONFIG = False
# ADV CONFIGURATION #########################################################
SITE_PACKAGE_PATH = ''
#############################################################################

**Comment by anjuthanmj** [comment]
@alvaroreig As @JosemyDuarte  suggested, if I add any tag to the pocket article, the plugin pulls that article. If there is not tag attached, then returns no article error. Is it possible to fix this?

**Comment by alvaroreig** [comment]
Hi everyone,

@JosemyDuarte you mention tld, but that part of the code was removed. Please upgrade to the latest version (2.7.5), try again and post your complete configuration.

@anjuthanmj please confirm that you are using the latest version. I will try to find a moment this weekend to replicate the problem with your configuration. It seems that the INCLUDE_UNTAGGED = True is not working properly with empty TAGS. In the meantime you can use SECTIONS_BY_DOMAIN = True to read your articles

@hdavidzhu please confirm that you are using the latest version and post your complete configuration.

**Comment by JosemyDuarte** [comment]
My bad @alvaroreig, you are right, I meant SECTIONS_BY_DOMAIN, not TLD. I think I'm running latest version already.

```
TAGS = [] # [] or ['tag1', 'tag2']
TAGS_EXCEPTIONS = [] # [] or ['tag3', 'tag4']
URL_KEYWORD_EXCEPTIONS = [] # [] or ['keyword1', 'keyword2']
SECTIONS_BY_DOMAIN = True
INCLUDE_UNTAGGED = True
ARCHIVE_DOWNLOADED = True
MAX_ARTICLES = 1000
OLDEST_ARTICLE = 2
SORT_METHOD = 'newest'
SORT_WITHIN_TAG_BY_TITLE = False
TO_PULL = 'unread'
TITLE_WITH_TAGS = False
ALLOW_DUPLICATES = False
USE_GLOBAL_CONFIG = False
```

**Comment by anjuthanmj** [comment]
> please confirm that you are using the latest version

@alvaroreig  Thanks a lot for your response. I downloaded the file from the installation instruction in the readme page. Should I download from that link or from the release tag section?

**Comment by JosemyDuarte** [comment]
@alvaroreig changed my settings to 

```
TAGS = [] # [] or ['tag1', 'tag2']
TAGS_EXCEPTIONS = [] # [] or ['tag3', 'tag4']
URL_KEYWORD_EXCEPTIONS = [] # [] or ['keyword1', 'keyword2']
SECTIONS_BY_DOMAIN = False
INCLUDE_UNTAGGED = True
ARCHIVE_DOWNLOADED = True
MAX_ARTICLES = 1000
OLDEST_ARTICLE = 365
SORT_METHOD = 'newest'
SORT_WITHIN_TAG_BY_TITLE = False
TO_PULL = 'unread'
TITLE_WITH_TAGS = False
ALLOW_DUPLICATES = False
USE_GLOBAL_CONFIG = False
```
But it keeps failing to find articles. `SECTIONS_BY_DOMAIN` keeps working correctly.

**Comment by alvaroreig** [comment]
I think I fixed it, you can download latest version v2.7.6 and let me know if you still have problems @JosemyDuarte @anjuthanmj @hdavidzhu
